### ISSUE TITLE
useOnDocument registered events dont work in storybook

### ISSUE THREAD

**Issue by DustinJSilk** [literalpie/storybook-framework-qwik#51]
Registering an event with useOnDocument in a component works when running in a Qwik app, however, when running that same component in storybook, the events no longer register and fire.

Such as: 

```tsx
useOnDocument('mouseup', $(() => console.log('mouseup')))
useOnDocument('mousedown', $(() => console.log('mousedown')))
```

**Comment by literalpie** [comment]
I believe this is a Qwik issue with Dev (non-SSR) mode. I created an issue in the Qwik repo

**Comment by literalpie** [comment]
Hi, I believe this is fixed in qwik 0.105.0!
